operating/manage-user-access/authentication-provider-structure.adoc

```diff
@@ -10,7 +10,7 @@ toc::[]
 
 An authentication provider connects to a third-party source of user identity (for example, an identity provider, or IdP). An authentication provider gets the user identity from IdP, issues a token based on this identity, and returns the token to the caller. The mentioned token is then sent within the user interface or API calls and allows {product-title-short} to authorize the user.
 
-You can set up your own IdP after provisioning by navigating to *Operating* -> *Managing user access* and selecting the appropriate module. 
+Navigate to Operating -> Managing user access, and select the appropriate module to set up your own IDP immediately after provisioning.
```

Comment:
`Operating -> Managing user access`

Reply:
Per Style Guide, I think this is bold and not backticks. Backticks are normally for objects or items and this is a menu choice on the web page (unless I'm misunderstanding).